packages/runtime/container-runtime-definitions/src/containerRuntime.ts

```diff
@@ -22,18 +22,10 @@ import {
 	FlushMode,
 	IContainerRuntimeBase,
 	IContainerRuntimeBaseEvents,
-	IDataStore,
 	IFluidDataStoreContextDetached,
 	IProvideFluidDataStoreRegistry,
 } from "@fluidframework/runtime-definitions";
 
-/**
- * @deprecated Not necessary if consumers add a new dataStore to the container by storing its handle.
- */
-export interface IDataStoreWithBindToContext_Deprecated extends IDataStore {
-	fluidDataStoreChannel?: { bindToContext?(): void };
```

Comment:
Wow this was more brittle than I realized.  `DataStore.fluidDataStoreChannel` is `IFluidDataStoreChannel` and we already removed `bindToContext` off that.  So we're totally depending on the concrete implementation that shows up there being a `FluidDataStoreRuntime`.  Did I get that right?

Reply:
Yes, that was the idea right. To remove it from interfaces but keep in the class implementations while we had this deprecated interface.